proposals/exp-show-target.md

```diff
@@ -0,0 +1,210 @@
+- Enhancement Proposal PR: (leave this empty)
+- Contributors: dberenbaum, (add your Github handle)
+
+# Summary
+
+The `dvc exp show` command shows a comparison of the results of different
+experiments along with the parameters and metrics values for each experiment.
+The output currently includes all values that are defined in `params.yaml` or
+any other referenced parameters files in any DVC stage. Output also includes all
+metrics in any referenced metrics files in any DVC stage. This proposal allows
```

Comment:
Took me a bit to understand that question. Maybe this is slightly clearer:
```suggestion
any other referenced parameters files in any DVC stage. It also includes all
values in any referenced metrics files in any DVC stage. This proposal allows
```

Reply:
I broke this into bullets. Hopefully it's clearer.